app/helpers/application_helper.rb

```diff
@@ -6,8 +6,8 @@ def knowledge_base_url
     'http://openstax.force.com/support/articles/FAQ/Can-t-log-in-to-your-OpenStax-account'
   end
 
-  def openstax_url
-    'https://openstax.org'
+  def logo_url
+    Rails.application.secrets.openstax_url
```

Comment:
Please add an entry to secrets.yml.example file?

Reply:
Done